docs/content/docs/build/data-modeling.md

```diff
@@ -59,15 +68,30 @@ const newCell = await fluidContainer.create(SharedCell); // Create a new SharedC
 const newMap = await fluidContainer.create(SharedMap); // Create a new SharedMap
 ```
 
+{{% callout tip %}}
+Another way to think about `initialObjects` and dynamic objects is as follows:
+
+With `initialObjects`, you're telling Fluid both the type of the object *and* the key you'll use to later retrieve the
+object. This is statically defined, so Fluid can create the object for you and ensure it's always available via the key
+your code defined.
+
+On the other hand, with dynamic objects, you're telling Fluid what object types it can create, but that's all. When your code creates a dynamic object using `container.create`, that objects is in-memory only. If you want to load that Fluid object again later, your code must store a reference to it within another Fluid object. In a
+sense, you're defining the "key" to access that data again later, just as you did with `initialObjects`, but you define
+it dynamically at runtime.
+
+{{% /callout %}}
+
 ### Using handles to store and retrieve Fluid objects
 
-All shared objects have a `handle` property that can be used to store and retrieve them from other shared objects. Objects created dynamically must be stored before they are collaborative. As you will see below, the act of storing a handle is what links the new dynamic object to the underlying data model and is how other clients learn that it exists.
+All Fluid objects have a `handle` property that can be used to store and retrieve them from other Fluid objects. Objects created dynamically must be stored before they are collaborative. As you will see below, the act of storing a handle is what links the new dynamic object to the underlying data model and is how other clients learn that it exists.
 
-Dynamically created objects need to be stored on an already connected shared object, so the most common case is to store them in an `initialObject`, because `initialObjects` are connected on creation. However, you can also store dynamic objects in other connected dynamic objects. In this sense shared objects are arbitrarily nestable.
+Dynamically created objects need to be stored on an already connected Fluid object, so the most common case is to store references to them in an initial object property, because initial objects are connected on creation. However, you can also store dynamic objects in other connected dynamic objects. In this sense Fluid objects are arbitrarily nestable. But the hierarchy of referenced objects must rest in an initial object that holds the references to the first level of dynamic objects. So, there is always at least one initial object in every container.
```

Comment:
```suggestion
Dynamically created objects need to be stored on an already connected shared object, so the most common case is to store references to them in an initial object property, because initial objects are connected on creation. However, you can also store dynamic objects in other connected dynamic objects. In this sense shared objects are arbitrarily nestable. But the hierarchy of referenced objects must rest in an initial object that holds the references to the first level of dynamic objects. So, there is always at least one initial object in every container.
```